client/webserver/locales/en-us.go

```diff
@@ -106,134 +106,135 @@ var EnUS = map[string]string{
+	"Offering":                               "Offering",
+	"Asking":                                 "Asking",
+	"Fees":                                   "Fees",
+	"order_fees_tooltip":                     "On-chain transaction fees, typically collected by the miner. Decred DEX collects no trading fees.",
+	"Matches":                                "Matches",
+	"Match ID":                               "Match ID",
+	"Time":                                   "Time",
+	"ago":                                    "ago",
+	"Cancellation":                           "Cancellation",
+	"Order Portion":                          "Order Portion",
+	"you":                                    "you",
+	"them":                                   "them",
+	"Redemption":                             "Redemption",
+	"Refund":                                 "Refund",
+	"Funding Coins":                          "Funding Coins",
+	"Exchanges":                              "Exchanges",
+	"apply":                                  "apply",
+	"Assets":                                 "Assets",
+	"Trade":                                  "Trade",
+	"Set App Password":                       "Set App Password",
+	"reg_set_app_pw_msg":                     "Set your app password. This password will protect your DEX account keys and connected wallets.",
+	"Password Again":                         "Password Again",
+	"Add a DEX":                              "Add a DEX",
+	"Pick a server":                          "Pick a server",
+	"reg_ssl_needed":                         "Looks like we don't have an SSL certificate for this DEX. Add the server's certificate to continue.",
+	"Dark Mode":                              "Dark Mode",
+	"Show pop-up notifications":              "Show pop-up notifications",
+	"Account ID":                             "Account ID",
+	"Export Account":                         "Export Account",
+	"simultaneous_servers_msg":               "The Decred DEX Client supports simultaneous use of any number of DEX servers.",
+	"Change App Password":                    "Change App Password",
+	"Build ID":                               "Build ID",
+	"Connect":                                "Connect",
+	"Withdraw":                               "Withdraw",
+	"Send":                                   "Send",
+	"Deposit":                                "Deposit",
+	"Lock":                                   "Lock",
+	"New Deposit Address":                    "New Deposit Address",
+	"Address":                                "Address",
+	"Amount":                                 "Amount",
+	"Authorize send with your app password.": "Authorize send with your app password.",
+	"Reconfigure":                            "Reconfigure",
+	"pw_change_instructions":                 "Changing the password below does not change the password for your wallet software. Use this form to update the DEX client after you have changed your password with the wallet software directly.",
```

Comment:
I'm not sure what going on with the formatting here. This new formatting seems to make sense, but if I'm on master and I run `gofmt`, this isn't what I get.

Did you make changes to this file? If you added entries, please put them at the bottom. If you updated existing entries, see if you can do it without changing the indentation everywhere.

Reply:
Just edited "Authorize the withdraw" to "Authorize send" which is the longest string on that map.